.github/workflows/rust.yml

```
@@ -23,37 +23,60 @@ env:
+# (/opt/libtorch), so there is no x86 `download-libtorch` (no aarch64 zip exists).
+#
+# AWS creds from the OIDC step are passed into the container by name (`-e VAR`
+# with no value forwards the host value) so the image's sccache reaches S3.
+#
+# The merge-gate `build` job stays on ubuntu-latest for now (arm64 default-feature
+# test validation is not green yet) — see its inline note.
 jobs:
   clippy:
     name: Clippy
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, linux, arm64, graviton, hyprstream-merge-gate]
```

actionlint (1.7.12)

[error] label "graviton" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)